install/docker-extension/ui/src/components/ExtensionComponent/ExtensionComponent.js

```diff
@@ -165,40 +172,40 @@ const ExtensionsComponent = () => {
           </ExtensionWrapper>
           <ExtensionWrapper>
             <div>
-              <CustomTypography sx={{ marginBottom: "1rem" }}>Deploy a Service Mesh</CustomTypography>
+              <Typography sx={{ marginBottom: "1rem", color:"white" }}>Deploy a Service Mesh</Typography>
```

Comment:
Does the Docker theme not have the light color defined already?

Reply:
It does, i had to set it to white cause we are using a dark background. After switching to a lighter background for dark mode we can remove this.

Reply:
ok